CodeAssignment_MVVM_Combine/Sources/Models/Presentations/RepositoriesForView.swift

```diff
@@ -0,0 +1,44 @@
+//
+//  RepositoryForView.swift
+//  CodeAssignment_MVVM_Combine
+//
+//  Created by gyusoku.i on 2024/05/13.
+//
+
+import Foundation
+
+/** ViewControllerに渡す用のstruct Model。VCで表示させるViewに適用させるために、新しくModelを作成
+ - Viewに表示するだけのModelを作ることで、APIを叩くときに余計にCoding Keysを使わなくてもいいし、テストもしやすくなるというメリットがある。
+*/
+struct Repositories {
+    /// 検索でマッチされたリポジトリの数
+    let totalCount: Int?
+    /// リポジトリの詳細データが入っている配列形
+    // 個々のリポジトリの情報が入っている配列
+    let items: [Repository]
+    
+    struct Repository {
+        var owner: RepositoryUser
+        var name: String?
+        var description: String?
+        var language: String?
+        var stargazersCount: Int?
+
+        struct RepositoryUser {
+            var login: String?
+            var avatarUrl: String?
+        }
+    }
+    
+    init(totalCount: Int?, repositories: RepositoriesResponse) {
+        self.totalCount = totalCount
```

Comment:
https://github.com/KyusokLee/CodeAssignment_MVVM_Combine/pull/3#discussion_r1598162149
ここで言ってたのはこういうことです！

```suggestion
    init(repositories: RepositoriesResponse) {
        self.totalCount = repositories.totalCount
```

Reply:
@Ogijun2018 
https://github.com/KyusokLee/CodeAssignment_MVVM_Combine/pull/3#discussion_r1600923598
https://github.com/KyusokLee/CodeAssignment_MVVM_Combine/pull/3#discussion_r1600924516
↑のコメントも合わせて[21c2ccc]にてコード修正しました！ご確認、お願いします！🙇